sympy/solvers/ode.py

```diff
@@ -1278,31 +1278,34 @@ def _test_term(coeff, order):
 
 def classify_sysode(eq, funcs=None, **kwargs):
     r"""
-    Returns a list of parameters defining the system of ordinary differential equations
-    in form of dictionary. The dictionary returned is further used in
-    :py:meth:`~sympy.solvers.ode.dsolve` for solving system of ODEs.
-
-    It returns seven parameters, first parameter is 'is_linear' which tells
-    whether given equation is linear or Non-linear based on the coefficients
-    of the functions of equations, i.e, coefficients of x, diff(x,t), diff(x,t,t) etc
-    which itself is a parameter named 'func_coeff'.
-    If the coefficient is constant, then the equation is said to be linear and 'is_linear'
-    returns True otherwise 'is_linear' returns False.
-    Second parameter is order of equation, it provides information about order
-    of differential equations provided. The third parameters is the number of equation
-    in the system.
-    The forth parameter is 'func_coeff' which has list of coefficient of all function and its
-    differentials. Each is described by three elements, first is equation number,
-    second is function and third is order of function whose coefficients we are calculating.
-    The other two parameters are the equations of ODEs and its functions.
-    The other parameter which tells whether given system of ODEs are solvable by current
-    solving engine. If it returns a type the it is solvable and result can be obtained
-    using dsolve otherwise None is returned.
-
-    The type returned is just a number assigned to set of each equation classified under
-    common characteristics based on linearity, number of equation and order. Similarly
-    there are different sets based on above parameters. The type number sequence implemented
-    is same as is given references.
+    Returns a dictionary of parameter names and values that define the system
+    of ordinary differential equations in ``eq``.
+    The parameters are further used in
+    :py:meth:`~sympy.solvers.ode.dsolve` for solving that system.
+
+    The parameter names and values are:
+
+    'is_linear' (boolean), which tells whether the given system is linear.
+
+    'func' (list) contains the :py:class:`~sympy.core.function.Function`s that
+    appear with a derivative in the ODE, i.e. those that we are trying to solve
+    the ODE for.
+
+    'order' (dict) with the maximum derivative for each element of the 'func'
```

Comment:
Yes. It's a list in `dsolve` (which might be wrong actually), but it is a dict here.
It's doing `order = dict` below, and `matching_hints['order'] = order` to set the result.